dev/com.ibm.ws.security.saml.websso.2.0/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -134,6 +134,9 @@ nameIDFormat.CUSTOMIZE=Customized Name ID Format.
 customizeNameIDFormat=Customized XML name space of name id format
 customizeNameIDFormat.desc=Specifies the customized URI reference corresponding to a name identifier format that is not defined in the SAML core specification.
 
+disableIRCookie=Disable Initial Request cookie creation
+disableIRCookie.desc=Do not create the initial request cookie if there are too many authentications requests from Service Provider to IdP as a result of the application SSO setup.
```

Comment:
Is it possible to say what constitutes "too many requests" ? What is the threshold? Or is it that the developer would set this attribute if they are receiving too many requests?

Reply:
This is not default behavior and developers would set this attribute if they are seeing more messages from SP (more than two at a time) than the responses from the SAML IdP and this leads to WASInitialRequest... cookie buildup  over the time and finally exceeds browser header size limit (resulting in exceeded the header size limit errors)